plugins/text-freetype2/text-freetype2.c

```diff
@@ -130,6 +130,12 @@ static obs_properties_t *ft2_source_properties(void *unused)
 	obs_properties_add_text(props, "text",
 		obs_module_text("Text"), OBS_TEXT_MULTILINE);
 
+	obs_property_t *alignmentProp = obs_properties_add_list(props, "alignment",
+		obs_module_text("Alignment"), OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_INT);
+	obs_property_list_add_int(alignmentProp, "Left", TEXT_ALIGNMENT_LEFT);
+	obs_property_list_add_int(alignmentProp, "Center", TEXT_ALIGNMENT_CENTER);
+	obs_property_list_add_int(alignmentProp, "Right", TEXT_ALIGNMENT_RIGHT);
+
```

Comment:
This block of text goes over the column limit (80), would probably be better like this:

```C
	obs_property_t *alignmentProp = obs_properties_add_list(props,
			"alignment", obs_module_text("Alignment"),
			OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_INT);
	obs_property_list_add_int(alignmentProp, "Left",
			TEXT_ALIGNMENT_LEFT);
	obs_property_list_add_int(alignmentProp, "Center",
			TEXT_ALIGNMENT_CENTER);
	obs_property_list_add_int(alignmentProp, "Right",
			TEXT_ALIGNMENT_RIGHT);
```